static/sass/mq.css

```diff
@@ -2528,3 +2528,11 @@ html[xmlns] .slides { display: block; }
 
 @viewport {
   width: device-width; }
+
+
+.element-4 ul,
```

Comment:
We support multiple devices so this should be moved into one of the applicable media blocks like

```scss
@media (min-width: px2em( ... )) {

}
```